ui/scss/_gui.scss

```diff
@@ -7,7 +7,7 @@ html
 body
 {
   color: var(--text-color);
-  font-family: 'Source Sans Pro', sans-serif;
+  font-family: 'Roboto', sans-serif;
```

Comment:
Bold move. Is this to move us inline with MD? Or do you just think this is appropriate font?

Reply:
lbry it's already usign `MD` so yes I think it's  appropriate.

Reply:
This PR isn't dropping `MD`, is doing some enhancement,
this would probably change when we implement the @nizuka01 `design`